avail/src/main/kotlin/avail/compiler/splitter/MessageSplitter.kt

```diff
@@ -1064,16 +1249,27 @@ constructor(messageName: A_String) {
 				"token or a group of lowercase tokens"))
 		{
 			group = group.applyCaseInsensitive()
+			// Rebuild the group, but including the '~'.
+			group = group.apply {
+				Group(
+					startInName,
+					currentPositionForEnd,  // Include the '~'.
+					beforeDagger.applyCaseInsensitive(),
+					hasDagger,
+					afterDagger.applyCaseInsensitive(),
+					maximumCardinality)
+			}
 		}
```

Comment:
Are you trying to overwrite `group` with the contents in the `apply` block? 
The `Group` constructed  inside the `apply` block is just being thrown away and `group` preserves its previous value.